fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/DepositAccountWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -1324,8 +1324,15 @@ public void updateMaturityDetails(Long depositAccountId, DepositAccountType depo
                         fdAccount.getOnAccountClosureId(), toSavingsId, "Apply maturity instructions", changes);
 
                 if (changes.get("reinvestedDepositId") != null) {
```

Comment:
This could  have been a variable to avoid calling changes.get in multiple places

Reply:
@oluexpert99  That's right but this is an old implementation from community. It's not my change. Thanks for spotting it